gapic/schema/wrappers.py

```diff
@@ -948,6 +948,21 @@ class Method:
     def __getattr__(self, name):
         return getattr(self.method_pb, name)
 
+    @property
+    def transport_safe_name(self) -> str:
+        # These names conflict with other methods in the transport.
+        # We don't want to disambiguate the names at the client level
+        # because the disambiguated name is less convenient and user friendly.
+        #
+        # Note: this should really be a class variable,
+        # but python 3.6 can't handle that.
+        TRANSPORT_UNSAFE_NAMES = {
```

Comment:
Looks like it is the second in a row disambiguating PR (the other one: https://github.com/googleapis/gapic-generator-python/pull/1178). Can we (should we) have some sort of centralized name-conflict resolution mechanism? I think it is fine for this PR to be as is, but in case more things like that happen, it would be  good to have a unified name-conflict resolution unility class or something like that. WDYT?

Reply:
Different names that refer to different data structures need to be disambiguated against different sets of names. It doesn't matter too much if there's a field named `create_channel`. I'd argue that there are costs to being too aggressive disambiguating names, and that it makes more sense to deal with them in the `Method` or `Field` class when collisions occur.

Reply:
I agree that they have to be disamgiguated against different sets of names, but that set of names can be a standardized parameter to the generic disambiguating mechanism. Basically some mechanism though which all names go through, and get disambiguated depending on the context. But yeah, this is beyond the scope of this PR and we have many more important tasks to work on.